CHANGELOG.md

```diff
@@ -6,6 +6,7 @@
   * Extend match macros to accept a plug and options
   * Make path parameters available in `conn.params`
   * Add `:init_opts` option to forward macro for plug options
+  * Add `path_params` to access path params apart from `params`
```

Comment:
Maybe we should say something like

> Add the `:path_params` field to `Plug.Conn` to access path params apart from the `params` field

so it's clear that we are adding this to the struct. Wdyt?